airbyte-integrations/connectors/destination-bigquery-denormalized/src/test/java/io/airbyte/integrations/destination/bigquery/BigQueryDenormalizedUtilsTest.java

```diff
@@ -206,6 +211,24 @@ void testSchemaWithNestedDatetimeInsideNullObject() {
 
   }
 
+  @Test
+  public void testEmittedAtTimeConversion() {
+    // in this test we don't care about json schema and naming resolver, so leave it as a null
+    final JsonNode jsonNodeSchema = getSchema();
```

Comment:
could we consider to use mock instead of instance?

Reply:
Just forgot to remove the comment, it's not applicable for now.
Record formatted requires a valid schema for now.
Removed obsolete comment. Thanks